docs/orchestration.asciidoc

```diff
@@ -116,8 +116,17 @@ If an Elasticsearch node holds the only copy of a shard, this shard becomes unav
 
 * Elasticsearch `Pods` may stay `Pending` during a rolling upgrade if the Kubernetes scheduler cannot re-schedule them back. This is especially important when using local `PersistentVolumes`. If the Kubernetes node bound to a local `PersistentVolume` does not have enough capacity to host an upgraded `Pod` which was temporarily removed, that `Pod` will stay Pending.
 
-* Rolling upgrades can only make progress if the Elasticsearch cluster health is green. It is risky to attempt upgrading a cluster in the yellow state as some shards could become completely unavailable and degrade the cluster health to red. ECK takes the cautionary approach of waiting for green health before progressing but advanced users may force an upgrade by manually deleting `Pods` themselves. The deleted `Pods` will be automatically recreated at the latest revision. There are two exceptions to this rule:
+* Rolling upgrades can only make progress if the Elasticsearch cluster health is green. ECK might also make progress if the cluster health is yellow under the following conditions:
```

Comment:
"ECK might" should be "ECK will"?

Also I wonder about the "only" in this sentence:
"Rolling upgrades can only make progress if the Elasticsearch cluster health is green."
since we immediately give other situations where it will make progress if it is not green. Maybe, "In most situations, rolling upgrades will only make progress if the Elasticsearch cluster health is green"?